.github/workflows/push_images.yml

```diff
@@ -0,0 +1,39 @@
+name: ci
+
+on:
+  push:
+    branches:
+      - 'feature/official_steamcmd'
```

Comment:
Change this `master` before merging

Reply:
Correct me if I'm wrong: as of now we only push new images by tagging and releasing the tag on GitHub.
Do we want to push new images on every `master` update and lose the tag-oriented workflow?

This means we will only have 3 tags without an history: `latest` (`vanilla`), `wine6` and `wine7`.

Reply:
Yes.


(is it bad?)

Reply:
I don't think we should aim for stability/consistency because the whole thing is subject to whatever changes they push to the game server we pull.
I'm also pretty sure that you don't have the certainty that old versions of game clients are allowed so you would have to stay on the latest version anyway (e.g. we can lock server version + container behaviour and not introduce braking changes but they can just release a new server version and nobody will be able to join our server because their game clients can just auto update).

I guess we can just decide to version majors (e.g. vanilla-v2) in case we want to introduce changes to the exposed docker features.. but then again, why would we be forced to change the EXPOSEd port or the volume path?

Reply:
> Correct me if I'm wrong: as of now we only push new images by tagging and releasing the tag on GitHub. Do we want to push new images on every `master` update and lose the tag-oriented workflow?
> 
> This means we will only have 3 tags without an history: `latest` (`vanilla`), `wine6` and `wine7`.

This seems the generally accepted behaviour/pattern. Also nobody would ever want to pull any of the older tagged versions so they would just pollute the dockerhub page.

A "tag" in the dockerhub world is just a channel and points to the latest version. The naming is quite retarted but I get why it's used like this. Docker seems also to push this narrative because of the way it renders "last pushed by" on tags so I would keep this convention and just push a different version on the tagname if we really break the usage/behaviour of the image and we would like to keep the old behaviour too (although I fail to see which conditions would enable such scenario).